### PR TITLE
Used wrong method to query world border size

### LINKED ISSUE #1225: system_info('world_size') either incorrect or misleading
The `system_info` option `world_size` is documented as:
`world_size` - Returns size of the world where at this distance from `world_center` world border appears.

But with the world border set to 400 this returns 29999984.
image

Either `world_size` is returning the incorrect value, or its documentation needs to be changed to something along the lines of:
`world_size` - Returns size of the world where at this distance from `world_center` world border appears **by default**.

### PR DESCRIPTION
**Pull request by Ghoulboy78**
Fixes #1225

### CODE REVIEW

**Comment by ch-yx** [comment]
could you rename it to world_border_size btw?

**Comment by James103** [comment]
> could you rename it to world_border_size btw?

I opened #1237 so that you can consider leaving the old behavior as `world_max_size`, in addition to adding in the new behavior under `world_border_size`.

**Comment by Ghoulboy78** [comment]
I'm not gonna do the `world_max_size` cos it's till being discussed as of now, but I will rename to `world_border_size`, yeah.

**Comment by altrisi** [comment]
Don't. Do. That.

You'll break absolutely every app that has used those. Deprecate them at most.

Also I believe in vanilla (with mods for sure) you can't configure the worldborder per dimension, at most it may get multiplied for the nether? idrk, may be wrong.

**Comment by ch-yx** [comment]
> Don't. Do. That.
> 
> You'll break absolutely every app that has used those. Deprecate them at most.
> 
> Also I believe in vanilla (with mods for sure) you can't configure the worldborder per dimension, at most it may get multiplied for the nether? idrk, may be wrong.

@altrisi cannot agree. 

that is broken since the begining. 
cannot see "every app that has used those".

**Comment by altrisi** [comment]
No. Only worldborder was broken.

**Comment by ch-yx** [comment]
> No. Only worldborder was broken.

and it is to fix worldborder.

**Comment by ch-yx** [comment]
ahh. he changed that into `dimension_xxx` now.


last time i saw it was to change world_border_size and world_max_size only

**Comment by Ghoulboy78** [comment]
> Don't. Do. That.
> 
> You'll break absolutely every app that has used those. Deprecate them at most.

I'll. do. that. in. stead

**Comment by Ghoulboy78** [comment]
tho tbh the feature has been out for like 1 week. How many people will have used it?

**Comment by gnembon** [comment]
add `world_max_size` and fix current behaviour of `world_size`. Beware that the area's getSize() is a diameter, not a radius, so you would need to half it.

concetually world border IS the world size. and world==dimension. so don't worry about renaming.

**Comment by Ghoulboy78** [comment]
ok done

**Comment by altrisi** [comment]
Still renamed bottom and top 😢.
It hasn't existed for a week, it has existed since [the end of november](https://github.com/gnembon/fabric-carpet/releases/tag/1.4.56).

**Comment by gnembon** [comment]
yeah, no need to rename `top` and `bottom` especially that it doesn't fixes the reported bug. I can merge it and rename it back, cause I see no reason of changing it.

File: docs/scarpet/api/Auxiliary.md [hunk 1]
```diff
@@ -697,10 +697,11 @@ Available options in the scarpet app space:
   * `world_gamerules` - returns all gamerules in a map form (`rule`->`value`). Like carpet rules, values are returned as strings, so you can use appropriate value conversions using `bool()` or `number()` to convert them to other values. Gamerules are read-only to discourage app programmers to mess up with the settings intentionally applied by server admins. Isn't that just super annoying when a datapack messes up with your gamerule settings? It is still possible to change them though using `run('gamerule ...`.
   * `world_spawn_point` - world spawn point in the overworld dimension
   * `world_time` - Returns dimension-specific tick counter.
-  * `world_top` - Returns current dimensions' topmost Y value where one can place blocks.
-  * `world_bottom` - Returns current dimensions' bottommost Y value where one can place blocks.
+  * `world_max_height` - Returns current dimensions' topmost Y value where one can place blocks.
```
Comment (gnembon) [inline docs/scarpet/api/Auxiliary.md:700]:
```suggestion
  * `world_top` - Returns current dimensions' topmost Y value where one can place blocks.
```

File: docs/scarpet/api/Auxiliary.md [hunk 2]
```diff
@@ -698,9 +698,10 @@ Available options in the scarpet app space:
   * `world_spawn_point` - world spawn point in the overworld dimension
   * `world_time` - Returns dimension-specific tick counter.
   * `world_top` - Returns current dimensions' topmost Y value where one can place blocks.
-  * `world_bottom` - Returns current dimensions' bottommost Y value where one can place blocks.
+  * `world_min_height` - Returns current dimensions' bottommost Y value where one can place blocks.
```
Comment (gnembon) [inline docs/scarpet/api/Auxiliary.md:701]:
```suggestion
  * `world_bottom` - Returns current dimensions' bottommost Y value where one can place blocks.
```

File: src/main/java/carpet/script/utils/SystemInfo.java [hunk 3]
```diff
@@ -65,16 +65,18 @@
             return ListValue.of(NumericValue.of(prop.getSpawnX()), NumericValue.of(prop.getSpawnY()), NumericValue.of(prop.getSpawnZ()));
         });
 
-        put("world_bottom", c-> new NumericValue(c.s.getWorld().getBottomY()));
+        put("world_min_height", c-> new NumericValue(c.s.getWorld().getBottomY()));
```
Comment (gnembon) [inline src/main/java/carpet/script/utils/SystemInfo.java:68]:
```suggestion
        put("world_bottom", c-> new NumericValue(c.s.getWorld().getBottomY()));
```

File: src/main/java/carpet/script/utils/SystemInfo.java [hunk 4]
```diff
@@ -67,14 +67,16 @@
 
         put("world_bottom", c-> new NumericValue(c.s.getWorld().getBottomY()));
 
-        put("world_top", c-> new NumericValue(c.s.getWorld().getTopY()));
+        put("world_max_height", c-> new NumericValue(c.s.getWorld().getTopY()));
```
Comment (gnembon) [inline src/main/java/carpet/script/utils/SystemInfo.java:70]:
```suggestion
        put("world_top", c-> new NumericValue(c.s.getWorld().getTopY()));
```

**Pull request merged**